rules/consistent-test-it.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const { getNodeName, isTestCase, isDescribe } = require('./util');
```

Comment:
cannot use destructuring due to node 4 support - see failing CI

Reply:
May be worth adding the [`es5/no-destructuring`](https://github.com/nkt/eslint-plugin-es5#list-of-supported-rules) rule to help prevent/communicate this.

Reply:
Opened https://github.com/jest-community/eslint-plugin-jest/pull/66 to address this.

Reply:
Yes, I see, Will be fixed